doc/source/serve/deployment-graph/model_composition.md

```diff
@@ -0,0 +1,273 @@
+(serve-model-composition)=
+
+# Model Composition: Deployment Graphs
+
+```{note}
+Note: This feature is in Alpha, so APIs are subject to change.
+```
+
+This section helps you:
+
+* connect your Ray Serve deployments together with the **deployment graph** API
+* serve your applications that use multi-model inference, ensemble models, ML model composition, or mixed business logic/model inference workloads
+* independently scale and configure each of your ML models and business logic steps
+
+Ray Serve's **deployment graph** API lets you compose your deployments together by describing how to route a request through your deployments. This is particularly useful if you're using ML model composition or mixing business logic and model inference in your application. You can encapsulate each of your models and each of your business logic steps in independent deployments. Then, you can connect these deployments together in a deployment graph.
+
+## Implicit Composition: Cross-Deployment Calls
+
+You can connect your models together in an implicit deployment graph by calling deployment methods from within other deployments. This lets you divide your application's steps (such as preprocessing, model inference, and post-processing) into independent deployments that can be independently scaled and configured. You can route requests to a single "driver" deployment that can partially process and/or forward the requests to other deployments by calling those deployments' methods.
+
+### ClassNodes
```

Comment:
I don't think we need to introduce ClassNode at all yet. The user only needs to think about deployments and handles in order to grok this section and accomplish basic model composition.

Reply:
Good call, I moved the subsection about `ClassNodes` into the next section (about the `Deployment Graph API`).